README.adoc

```diff
@@ -913,7 +913,7 @@ This rule should obviously be ignored if the complementing predicate
 
 === `comp` [[comp]]
 
-Leverage `comp` when doing so yields simpler code.
+Leveraging `comp` when doing so yields simpler code.
```

Comment:
The idea here is to suggest using `comp` when it makes sense. How is the new wording better? Perhaps "Consider using/prefer using` would make more sense here?

Reply:
'Leverage `comp` when doing so yields simpler code' isn't a grammatical English sentence, whereas the same sentence begun with 'Leveraging' is.

It seems the subject of this sentence is meant to be the act of using `comp`. Grammatically it's presented in the imperative mood 'leverage', which denotes a command. 

For a shorter more sandboxy example, 'Go to the store is a good idea' is a sentence that is gramatically divergent in the same way. 

For the act of leveraging `comp` to be the subject of the sentence, it needs to be changed to the gerund (as in the changed code). 

Remaining in the imperative mood with a verbiage change as you suggest is also a viable path. But to be a gramatically correct sentence it would need further changes. Starting with a stripped down version, perhaps 'Leverage/consider using/prefer using `comp` when doing so for simpler code.'

If the grammar case seems reasonable to you what is your preferred rendering?

Reply:
Got it. My remark was not so much about the grammar as about the general wording of the guideline itself. The way it's written it is way too generic and unclear, so I'm thinking it'd better to rephrase it completely. The current wording implies we're just showing some example, but what we should have a guideline illustrated by an example. I hope this makes sense to you.

Reply:
Sorry for the delay -- was on vacation and lost track.

What I'm hearing/inferring from you is we'd like to make these `comp` sections clearer by adding examples, and that whatever wording best supports that new structure is appropriate. I'll check for an open issue and add if not there

Reply:
^^ I see my first take was not right. 

From the current state of the style guide, I see that examples for these sections already exist. On reading your message over again I understand what you mean -- it is the guidelines for these sections that are missing and need to be written.

Reply:
Opened #191 to address

Reply:
I'm going to take the two small non-`partial`/`comp`-related aesthetic changes and open a new tiny PR with them rather than amending this branch (which I would otherwise do to avoid a backtracking commit on a tiny change).

I will close this when that's done

Reply:
Opened #192 to pop the non-`comp`/`partial` guideline creation stuff into its own branch